qcodes/dataset/dond/sweeps.py

```diff
@@ -57,6 +57,9 @@ def post_actions(self) -> ActionsT:
         """
         pass
 
+    def __repr__(self) -> str:
+        return f"Sweep of {self.param.full_name}"
+
```

Comment:
Either improve or remove this repr

Reply:
perhaps ``f"{self.__class__.__name__} of {self.param.full_name}"`` ?